pkgs/tools/graphics/xcur2png/default.nix

```
@@ -11,6 +19,18 @@ stdenv.mkDerivation rec {
     sha256 = "0858wn2p14bxpv9lvaz2bz1rk6zk0g8zgxf8iy595m8fqv4q2fya";
   };
 
+  patches = [
+    # implement malloc workaround to allow xcur2png to be cross-compiled
+    # https://www.linuxquestions.org/questions/linux-software-2/undefined-reference-to-%60rpl_malloc%27-587256/
+    ./malloc.diff
```

```suggestion
    # https://github.com/eworm-de/xcur2png/pull/3
    ./malloc.diff
```

Perhaps also submit the other patch? No idea why arch maintainers didn't do that.

---

Looks like the commit already exists on a separate branch https://github.com/eworm-de/xcur2png/commit/aa035462d950fab35d322cb87fd2f0d702251e82. Should we fetch the patch from there?

---

> Looks like the commit already exists on a separate branch [eworm-de/xcur2png@aa03546](https://github.com/eworm-de/xcur2png/commit/aa035462d950fab35d322cb87fd2f0d702251e82). Should we fetch the patch from there?

Yes